### ISSUE TITLE
Feature Request: Configurable duration

### ISSUE THREAD

**Issue by Shoob189** [hiddenkrypt/lancer-combat-banner#12]
I love the banners, but the start of turn banners stay on screen a little bit long for my play group. Would it be possible to add separate configurable duration for the start of turn banners and the start of round banners?

**Comment by hiddenkrypt** [comment]
Good idea! I'll keep this in mind as I work on the next major release.
